web/src/main/java/uk/ac/ebi/phenotype/web/controller/FileExportController.java

```diff
@@ -980,20 +969,20 @@ else if (fld.equals("symbol_gene")) {
 
 		JSONArray docs = json.getJSONObject("response").getJSONArray("docs");
 
-		List<String> rowData = new ArrayList();
+		List<String> rowData = new ArrayList<>();
 
 		rowData.add(
 				"Gene symbol\tHuman ortholog\tGene id\tGene name\tGene synonym\tProduction status\tPhenotype status\tPhenotype status link"); // column
 																																				// names
 
 		for (int i = 0; i < docs.size(); i++) {
-			List<String> data = new ArrayList();
+			List<String> data = new ArrayList<>();
 			JSONObject doc = docs.getJSONObject(i);
 
 			data.add(doc.getString("marker_symbol"));
 
 			if (doc.containsKey("human_gene_symbol")) {
-				List<String> hsynData = new ArrayList();
+				List<String> hsynData = new ArrayList<>();
 
 				JSONArray hs = doc.getJSONArray("human_gene_symbol");
```

Comment:
It might be too much work, but could we convert to use the SolrJ method for this?  These hardcoded field names seem to be common.